pyproject.toml

```
@@ -35,7 +36,8 @@ write_to = "src/_version.py"
 [project.optional-dependencies]
 tests = [
     "pytest>=5.4.3",
-    "hypothesis"
+    "hypothesis",
+    "NeSST>=1.0.0"
```

```suggestion
    "NeSST>=1.1.0"
```
this one makes use of endf python package for cross sections